BUILD

```diff
@@ -140,9 +159,12 @@ Dinit should generally build fine with no additional options, other than:
 
 Recommended options, supported by at least GCC and Clang, are:
  -Os       : optimise for size
- -fno-rtti : disable RTTI (run-time type information), it is not required by Dinit. However, on
-             some platforms such as Mac OS (and historically FreeBSD, IIRC), this prevents
-             exceptions working correctly.
+ -fno-rtti : disable RTTI (run-time type information) for some types to reduce output binary size.
+             RTTI is a feature of C++ that exposes (at runtime) information about the dynamic types
+             of objects. Dinit doesn't need RTTI for all types. However, on some platforms that use
+             Clang with libcxxrt as the C++ runtime library, this prevents Dinit from working
+             correctly (the known platforms exhibiting this problem are FreeBSD and macOS).
+             See "Special note for Clang (on systems using Libcxxrt C++ runtime)", below.
```

Comment:
If this is following from and directly related to the previous sentence, you use ';' as a joiner instead of a full stop.